src/actions/accountsActions.js

```diff
@@ -166,7 +166,7 @@ export const switchAccountAction = (accountId: string) => {
     if (account.type === ACCOUNT_TYPES.SMART_WALLET) {
       if (sdkInitialized) {
         await dispatch(connectSmartWalletAccountAction(accountId));
-        dispatch(setUserEnsIfEmptyAction());
+        dispatch(checkUserENSNameAction());
```

Comment:
So changed this a bit. It's not completely ran on startup as it's only ran in case of Archanova init and/or Etherspot import (executed during onboarding and Etherspot account missing scenarios).

Added these changes now:
- Removed from Archanova init action.
- Removed from Account switch action.
- Added to `loginAction` after a check if Etherspot Account exists.